cmd/ctl/pkg/renew/renew_test.go

```diff
@@ -75,7 +75,12 @@ func TestValidate(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			cmd := NewCmdRenew(genericclioptions.IOStreams{})
+			cmd := NewCmdRenew(genericclioptions.IOStreams{}, nil)
+
+			// This is normally registered in the main func. We add here to test
+			// against flags normally inherited.
```

Comment:
:+1: I wonder if we can instead instantiate the root command and then write each of these test cases as `rootCmd.Execute("renew", "xyz")` (i.e. as actual flags?) Let's not do that just now though :)

Reply:
Yes, now it's not in main we should be able to